ADAL/src/validation/ADAuthorityValidation.m

```diff
@@ -163,6 +164,12 @@ - (void)validateAuthority:(ADRequestParameters*)requestParams
     // Check for AAD or ADFS
     if ([ADHelpers isADFSInstanceURL:authorityURL])
     {
+        if (!validateAuthority)
+        {
+            completionBlock(NO, nil);
+            return;
+        }
+        
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3AS1131)